Sources/TokamakCore/Views/TextField.swift

```diff
@@ -52,8 +65,13 @@ public struct _TextFieldProxy {
 
   public init(_ subject: TextField<Text>) { self.subject = subject }
 
+  public func with(style: TextFieldStyle) -> Self {
+    _TextFieldProxy(TextField(_from: subject, textFieldStyle: style))
+  }
+
```

Comment:
If you apply suggestions above, there's no need for this function.

```suggestion
  public func with(style: TextFieldStyle) -> Self {
    _TextFieldProxy(TextField(_from: subject, textFieldStyle: style))
  }

```